index-singlecell.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: subsite-galaxy
+website: https://singlecell.usegalaxy.eu
+---
+
+![single cell Galaxy](/assets/media/logo_single_cell.svg){:.sc-intro-left}
+
+# Welcome to the world of Single Cell Omics
+{:.no_toc}
+
+The Sincle Cell Omics workbench is a comprehensive set of analysis tools and consolidated workflows. The workbench is based on the [Galaxy framework](https://galaxyproject.org/), 
+which guarantees simple access, easy extension, flexible adaption to personal and security needs, and sophisticated analyses independent of command-line knowledge.
+
+The current implementation comprises more than 20 bioinformatics tools dedicated to different research areas of single cell biology.
+
+This service is a joint project between different groups from the [Earlham Institute](http://www.earlham.ac.uk), the [EBI](https://www.ebi.ac.uk), [EMBL](https://gbcs.embl.de) the [Sorbonne University](http://drosophile.org/) and the [University of Freiburg](https://galaxyproject.eu/freiburg/).
```

Comment:
> the [Sorbonne University](http://drosophile.org/)

the [Sorbonne University](http://artbio.fr/)

Reply:
Changed that ...